### DISCUSSION TITLE
V4 compatibility

### DISCUSSION THREAD

**Discussion by backpack-operations** [Laravel-Backpack/community-forum#761] [Q&A (Help)]
Hell Team i just received this on email "DevTools, Editable-Columns, Figma-Template  . Will these 3 tools still support Backpack version 4 well?"

**Comment by pxpm** [comment]
Hey @backpack-operations 

Backpack v4 last commit (code change) was almost a year and half ago. What worked at that time still works, but we are more than 3000 commits ahead on Backpack v6 (CRUD alone), so there's a lot of functionality that's not present in v4. 
We will not push changes to v4 anymore, and **I don't think devtools or editable columns would work well with it**. 
Figma not 100% sure, but it's probable that it still works fine. 

Cheers
